src/renderer/screens/exchange/swap/Form/FromAccount.js

```diff
@@ -111,20 +121,28 @@ const FromAccount = ({
           value={currency}
           autoFocus={true}
           onChange={setCurrency}
-          isCurrencyDisabled={isCurrencyDisabled}
         />
       </Box>
       <Box>
         <Label mb={4} mt={25}>
-          <Trans i18nKey={`swap.form.from.account`} />
+          <Trans i18nKey={`swap.form.to.account`} />
```

Comment:
Does it have to be dynamic? (from <-> to). I haven't check the UI so I'm not sure but.

Reply:
It incidentally is the same wording, simply "Account" but you're right it should be "to"